Example/android/app/build.gradle

```diff
@@ -96,7 +96,7 @@ android {
     defaultConfig {
         applicationId "com.example"
         minSdkVersion 16
-        targetSdkVersion 22
+        targetSdkVersion 24
```

Comment:
Why is this necessary?

Reply:
Got build error of "Package com.example new target SDK 22 doesn't support runtime permissions but the old target SDK 24 does"... 
After deleting app/clearing cache "targetSdkVersion 22" is actually working. 👍